KeysPerSecond/src/me/roan/kps/Menu.java

```diff
@@ -204,11 +204,8 @@ protected static final void createMenu(){
 		});
 		sresetavg.addActionListener((e)->{
 			Main.n = 0;
-			Main.avg = 0;
-			Main.tmp = 0;
-		});
-		sresettot.addActionListener((e)->{

```

Comment:
The reset total funtion cannot be removed. It cannot be merged with the reset average.